ci/pipeline.yml

```diff
@@ -361,12 +393,23 @@ jobs:
       - put: google-cloud-run-buildpack-version
         params:
           file: instana-saas-version/release
+          
+  - name: update-ruby-dependencies-version-on-gem-update
+    max_in_flight: 1
+    plan:
+      - get: instana-ruby-gem
+        trigger: true
+      - put: instana-ruby-dependencies-version
+        params:
+          file: instana-ruby-gem/version
 
   - name: build-package-builder-image
     max_in_flight: 1
     plan:
       - get: cnb-package-builder-source
         trigger: true
+      - get: ruby-autotrace
```

Comment:
why do we get this here?

Reply:
This is downloading the shim code we use to boot the gem without it being included in the Gemfile. Its bundled into the buildpack package.